BUILD.bazel

```diff
@@ -367,7 +367,6 @@ cc_library(
         ":reference_element",
         ":reference_errors",
         ":reference_scope",
-        ":reference_sizes",
```

Comment:
Perhaps needs to be changed to `":reference_index"`? Due to some reason, Bazel isn't doing layering checks, but I think Blaze will fail on this.

Reply:
Done.